src/components/mypage/ReviewModal/index.tsx

```diff
@@ -0,0 +1,135 @@
+import type { ChangeEventHandler, ReactElement } from 'react'
+import { useState } from 'react'
+import { Styled } from './styled'
+import type {
+  ReviewModalProps,
+  EvaluateState,
+  EvaluateData,
+  ReviewState
+} from './types'
+
+const evaluateData: EvaluateData = [
+  {
+    state: 'smile',
+    text: '좋아요'
+  },
+  {
+    state: 'meh',
+    text: '보통이에요'
+  },
+  {
+    state: 'sad',
+    text: '별로에요'
+  }
+]
+
+const ReviewModal = ({
+  isOpen = true,
+  onClose,
+  onClick,
```

Comment:
```suggestion
  onConfirm,
```
여기두 onConfirm이 명확할 것 같아요!